torch/testing/_internal/torchbind_impls.py

```diff
@@ -30,3 +32,99 @@ def fake_queue_push(tq, x):
     @register_if_not("_TorchScriptTesting::queue_size")
     def fake_queue_size(tq):
         return tq.size()
+
```

Comment:
cleaner thing to do: make sure register_fake_operators gets run only once instead of if-then-else everywhere below